Fix build error:
    > Could not find org.scijava:pom-scijava:40.0.0.
      Searched in the following locations:
        - https://maven.scijava.org/content/groups/public/org/scijava/pom-scijava/40.0.0/pom-scijava-40.0.0.pom
      Required by:
          unspecified:unspecified:unspecified

Author: Samuel Moors (Vrije Universiteit Brussel)

diff -ur qupath-0.6.0.orig/settings.gradle.kts qupath-0.6.0/settings.gradle.kts
--- qupath-0.6.0.orig/settings.gradle.kts	2025-06-26 17:13:10.000000000 +0200
+++ qupath-0.6.0/settings.gradle.kts	2025-12-08 15:50:12.686124000 +0100
@@ -107,7 +107,11 @@
     }
 
     repositories {
-        maven("https://maven.scijava.org/content/groups/public/")
+        mavenCentral()
+        maven {
+            name = "SciJava"
+            url = uri("https://maven.scijava.org/content/groups/public")
+        }
     }
 
 }
